production/grafanacloud-install.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
```

Comment:
nit: We may want to consider going for full portability and using `#!/bin/sh`. It would mean a little bit of refactoring. Checking with `shellcheck --shell=sh` says the only change we would need to make is to drop `-o pipefail` which we don't currently use anywhere.

Reply:
Just wanted to mention `shellcheck` as well :slightly_smiling_face: